client/src/components/TermsAndConditionsModal.js

```diff
@@ -0,0 +1,308 @@
+          its use, nor liable for any viruses or other contamination of your
+          system nor for any delays, inaccuracies, incompleteness, errors, or
+          omissions arising out of your use of the TDM Calculator or with
+          respect to the material contained in the TDM Calculator. You
+          understand and agree that Your sole remedy against the City or Hack
+          for LA for loss or damage caused by any defect or failure of the TDM
+          Calculator, regardless of the form of action, whether in contract,
+          tort, including negligence, strict liability, or otherwise, shall be
+          the repair or replacement of the TDM Calculator to the extent feasible
+          as determined solely by the City. In no event shall the City or Hack
+          for LA be responsible to You or anyone else for, or have liability for
+          any special, indirect, incidental, or consequential damages
+          (including, without limitation, damages for loss of business profits
+          or changes to businesses costs) or lost data or downtime, however
+          caused, and on any theory of liability from the use of, or the
+          inability to use, the TDM Calculator, whether the data, and/or
+          formulas contained in the TDM Calculator are provided by the City or
+          Hack for LA, or another third party, even if the City has been advised
+          of the possibility of such damages.
+        </p>
+
+        <p className="smaller">
+          This Agreement and License shall be governed by the laws of the State
+          of California without regard to their conflicts of law provisions and
+          shall be effective as of the date that You are making use of the TDM
+          Calculator. Any action brought to enforce the Agreement and/or in
+          connection with this website shall be brought in either the state or
+          federal courts in the County of Los Angeles County, State of
+          California.
+        </p>
+
+        <p className="smaller">
+          By using the TDM Calculator, You hereby waive and release all claims,
+          responsibilities, liabilities, actions, damages, costs, and losses,
+          known and unknown, against the City and Hack for LA for Your use of
+          the TDM Calculator.
+        </p>
+
+        <p className="smaller">
+          Before making decisions using the information provided in this
+          application, contact City LADOT staff at{" "}
+          <a href="mailto:ladot.tdm@lacity.org" style={{ color: "blue" }}>
+            ladot.tdm@lacity.org
+          </a>{" "}
+          to confirm the validity of the data provided.
+        </p>
+      </div>
+
+      <div className={classes.modalActions}>
+        <Button
+          className={classes.declineBtn}
+          onClick={e => {
+            e.preventDefault();
+            window.location.href = "https://ladot.lacity.org/";
```

Comment:
Can you use `history.push("https://ladot.lacity.org/") here instead of window.location?

Reply:
Hi Claire, I tried subbing out window.location with history.push() but it just added on to the end of the URL. Is there like a history.clear() method that will reset the URL and push the new one?

![Screen Shot 2020-10-25 at 2 50 53 PM](https://user-images.githubusercontent.com/40730303/97120046-9349df00-16d1-11eb-9905-b06dc7e95f22.jpg)

Reply:
@jahpe777 i see. i don't have an immediate solution for this. just use window.location since we know that works for navigating outside our site. don't want you to spend too much time on this.

Reply:
Ok, thanks!